exporters/prometheus/exporter.go

```diff
@@ -469,8 +471,14 @@ func (c *collector) scopeInfo(scope instrumentation.Scope) (prometheus.Metric, e
 		return scopeInfo, nil
 	}
 
-	scopeInfo, err := createScopeInfoMetric(scope)
+	err, ok := c.scopeInfoErrs[scope]
+	if ok {
+		return nil, fmt.Errorf("cannot retrieve scope info metric: %w", err)
```

Comment:
How about returning some sentinel error instead and then checking for it here: 

https://github.com/open-telemetry/opentelemetry-go/blob/f87115c9586a3fc96bcfa8ad5a2b284c31836fe8/exporters/prometheus/exporter.go#L181-L185

so that we do not log the problem during each collection?

Reply:
Then we could also change `scopeInfoErrs: make(map[instrumentation.Scope]error)` to `scopeInfosInvalid: make(map[instrumentation.Scope]struct{})`

Reply:
Good idea. I actually used a `bool` instead of a `struct{}` in `scopeInfosInvalid`, since that's all we need.
I've also now been able to test that `otel.Handler()` only receives errors once.